staging/src/k8s.io/kubectl/pkg/cmd/create/create_job_test.go

```diff
@@ -193,6 +193,78 @@ func TestCreateJobFromCronJob(t *testing.T) {
 			}
 			job := o.createJobFromCronJob(tc.from)
 
+			if !apiequality.Semantic.DeepEqual(job, tc.expected) {
+				t.Errorf("expected:\n%#v\ngot:\n%#v", tc.expected, job)
+			}
+		})
+	}
+
+}
+
+func TestCreateJobFromJob(t *testing.T) {
+	jobName := "test-job"
+	suspended := true
+	labels := make(map[string]string)
+	labels["controller-uid"] = "test"
```

Comment:
We are trying to phase out these labels for the official Kubernetes ones so could you please use those instead?  

https://kubernetes.io/docs/reference/labels-annotations-taints/#batchkubernetesio-job-name